### ISSUE TITLE
screen usage on controllers

### ISSUE THREAD

**Issue by austinscreations** [OXRS-IO/OXRS-IO-WEBSITE#6]
might also be worth adding a page talking about controllers - but more specifically the usage of the screen and possible layouts

this would be in regards to the work moin has done both overall and with their recent suggestions from sumnerboy on discord.

any possible layout or data in regards to other configurations between input and output options that someone my build could / may be discussed.

**Comment by sumnerboy12** [comment]
Yep good idea. The plan is to provide a library for makers and firmware writers which makes accessing the screen easy and consistent. 

But that is optional of course and you can do whatever you like if needed. 

But a page explaining all this is a good idea.
